src/CarbonAware.CLI/CommandOptions.resx

```diff
@@ -129,6 +129,9 @@
   <data name="locationsDescription" xml:space="preserve">
     <value>A list of locations</value>
   </data>
+  <data name="rootCommandDesc" xml:space="preserve">

```

Comment:
`rootCommandDescription` to match the style of the other descriptors in the library